packages/flutter/test/widgets/animated_image_filtered_repaint_test.dart

```diff
@@ -0,0 +1,57 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ui';
+
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  testWidgets('ImageFiltered avoids repainting child as it animates', (WidgetTester tester) async {
+    RenderTestObject.paintCount = 0;
+    await tester.pumpWidget(
+      Container(
+        color: Colors.red,
+        child: ImageFiltered(
+          imageFilter: ImageFilter.blur(sigmaX: 5, sigmaY: 5),
+          child: const TestWidget(),
```

Comment:
Is there ever a reason that the compiler might decide to not honor the const? Is this any better than just creating it outside the calls to pumpWidget and reusing it manually? The count could then be non-static...

Reply:
No, `const` can't be ignored. However this isn't related to the the static on RenderTestObject. It will be the same with or without the const.

the static is on the render object, not the widget. If I made it an instance field I could look it up at runtime, but that's a lot of plumbing for a unit test

Reply:
Oh, yes, I missed that.

Reply:
We probably want it to be static anyway, since if something goes wrong and we re-created the render object we wouldn't want the count to be reset